### ISSUE TITLE
View SSL Cert and Encryption Details of Website

### ISSUE THREAD

**Issue by fictitiousexistence** [atlas-engineer/nyxt#2146]
**Is your feature request related to a problem? Please describe.**
Not related to any problem. All other browsers include a way to view a sites current ssl certificate along with the tls version and encryption methods in use. 

**Describe the solution you'd like**
A quick way to view a websites ssl cert and possibly the encryption in use. Similar to firefox/ chrome. 
Could be non fancy, like the output of "openssl s_client -connect example.com:443" or opening the cert in the systems built in cert viewer. 

**Describe alternatives you've considered**
Using openssl s_client -connect example.com:443" in a terminal

**Additional context**
Example image from firefox. The view certificate button lets you view more about the cert.
![firefox_ssl](https://user-images.githubusercontent.com/20089581/158284287-4932112e-f5fb-4cbb-a0d5-8f295f1caf5a.png)

**Comment by aartaka** [comment]
Indeed, could be useful! Seems that [WebKit allows](https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html#webkit-web-view-get-tls-info) inspecting the [certificate structure](https://www.freedesktop.org/software/gstreamer-sdk/data/docs/latest/gio/GTlsCertificate.html).

**Comment by fictitiousexistence** [comment]
My main use for this is work. I often have to update ssl certs and this is always a handy way to quickly confirm they were updated properly. Thanks!

**Comment by jmercouris** [comment]
There are other browsers that do this, this isn't a core focus of Nyxt.
